### ISSUE TITLE
Jupyter notebook execute the script but does not show outputs?

### ISSUE THREAD

**Issue by brafik12** [brafik12/raf#1]
Hello, I'm executing a script written in python on Jupyter notebook without any problem, but I'm facing an issue which is that I don't get results (outputs) in return?
`import pandas as pd
from numpy import mean
from numpy import std
from numpy import dstack
from keras.models import Sequential
from keras.layers import Dense
from keras.layers import Flatten
from keras.layers import Dropout
from keras.layers import LSTM
from keras.utils import to_categorical
from matplotlib import pyplot
from sklearn.model_selection import train_test_split

Data=pd.read_excel('C:\Desktop\Project\LSTM\Df.xlsx')

X = Data.iloc[:, [0,1,2,3,4,5,6,7]]

y= Data.iloc[:,[-1]]

X.Location=pd.Categorical(X.Location, ['Kh_miliana','Mostaganem','Chlef'])
X.Location=X.Location.cat.codes
#print(X.Location)

# convert the 'Date' column to datetime format 
X['Date']= pd.to_datetime(X['Date']) 

y.Treatment_status=pd.Categorical(y.Treatment_status, ['No treatment','Treatment applied'])
y.Treatment_status=y.Treatment_status.cat.codes
#print(y.Treatment_status)

# create training and testing vars
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3)
print (X_train.shape, y_train.shape)
print (X_test.shape, y_test.shape)

# fit and evaluate a model
def evaluate_model(X_train, y_train, X_test, y_test):
    verbose, epochs, batch_size = 0, 15, 64
    n_timesteps, n_features, n_outputs = X_train.shape[1], X_train.shape[2], y_train.shape[1]
    model = Sequential()
    model.add(LSTM(8, input_shape=(n_timesteps,n_features)))
    model.add(Dropout(0.5))
    model.add(Dense(2, activation='relu'))
    model.add(Dense(n_outputs, activation='softmax'))
    model.compile(loss='categorical_crossentropy', optimizer='adam', metrics=['accuracy'])
    
# fit network
    model.fit(X_train, y_train, epochs=epochs, batch_size=batch_size, verbose=verbose)

# evaluate model
    _, accuracy = model.evaluate(X_test, y_test, batch_size=batch_size, verbose=0)
    return accuracy

# summarize scores
def summarize_results(scores):
    print(scores)
    m, s = mean(scores), std(scores)
    print('Accuracy: %.3f%% (+/-%.3f)' % (m, s))
    

# run an experiment
def run_experiment(repeats=10):
# load data
    X_train, y_train, X_test, y_test = load_dataset(X,y)
# repeat experiment
    scores = list()
    for r in range(repeats):
        score = evaluate_model(X_train, y_train, X_test, y_test)
        score = score * 100.0
        print('>#%d: %.3f' % (r+1, score))
        scores.append(score)
    # summarize results
    summarize_results(scores)

# run the experiment
    run_experiment()

[Script.docx](https://github.com/brafik12/raf/files/4433568/Script.docx)
[Df.xlsx](https://github.com/brafik12/raf/files/4433570/Df.xlsx)
`
